src/java.base/share/classes/java/lang/Object.java

```diff
@@ -62,7 +63,7 @@
      * Constructs a new object.
      */
     @HotSpotIntrinsicCandidate
-    public Object() {}
+    public @Unique Object() {}
```

Comment:
Isn't this already in master? Please pull in master.

Reply:
The reason for the difference is that I had pushed these changes before my other PR was merged. I have pulled in master in my latest commit.